etl/jobs/clean-and-aggregate-monthly-nearest50-parameterised.py

```diff
@@ -0,0 +1,153 @@
+import sys
```

Comment:
What is the role of this script?  As its not part of any of the workflows, could be slightly confusing, maybe a comment at the top of the script or a note in the README to clarify which jobs form part of the workflow and which are just there for one-off data cleaning up or whatever?